### PR TITLE
Gidk 1nerfb selected hover test

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

**Review by aditi-tiwari17 — APPROVED** [review]

File: apps/web-giddh/src/app/ledger/ledger.component.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
     <!-- region ledger table-->
     <div class="ledg-head transactions-page mr-t15 position-relative">
         <!-- region ledger header -->
-        <div class="clearfix row mr-b15">
+        <div class="clearfix row mr-b15" [ngClass]="{'mr-b5' :  currentCompanyBranches?.length > 2}">
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/ledger/ledger.component.html:18]: Please check for scenario with company branches > 2 and user is switched to branch

File: apps/web-giddh/src/app/contact/contact.component.html [hunk 2]
```diff
@@ -93,7 +93,7 @@
                                 </sh-select>
                             </div>
                         </div>
-                        <div class="pull-left header-left">
+                        <div class="pull-left header-left" [ngClass]="{'on-company': currentCompanyBranches?.length <= 2 }">
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/contact/contact.component.html:96]: Please check for company with more than 2 branches and user is switched to company mode

**Review by prateek-giddh — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/ledger/ledger.component.html (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/ledger/ledger.component.html:18]: this mr-b5 class will be applied only when a company has a branch of more than 2 , working correctly now

**Review by prateek-giddh — APPROVED** [review]

**Pull request merged**